README.md

```diff
@@ -17,13 +17,13 @@ Why should you use this over other Kotlin bots:
 - Really easy to configure
 - Entirely modular, including config types and commands
 - Config types support providing a URL instead of a file name, allowing you to load configurations remotely
-- Command registration is automatic, just create an object which extends `Command("commandName")`
+- Command registration is automatic, just create an object which extends `BotCommand(name = "Example", description = "Example Command)`
```

Comment:
the syntax is incorrect here, you're missing a "

Reply:
Whops